charts/rancher-csp-adapter/0.1.0+up0.1.0-rc6/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Rancher CSP Adapter
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.22.0'
+  catalog.cattle.io/namespace: cattle-csp-adapter-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
+  catalog.cattle.io/rancher-version: '>= 2.6.6-0 <= 2.6.99-0'
```

Comment:
Change this to `'>= 2.6.6-0 < 2.7.0-0'`

Reply:
Yes, but I will submit a PR soon to fix these and stick to one convention. Some charts use < 2.6.99-0, others use <= 2.6.99-0 and this is problematic for cases where we set the Rancher dev version to 2.6.99 and need to do comparisons like we do for airgap image exporting (See [here](https://github.com/rancher/rancher/blob/68c6c1377b6d1dec9846193fd093a3b6bba8476e/pkg/image/charts.go#L205-L211)). So the < 2.7.0-0 would take care of this and also help stick to the same convention we use for the kube-version.

Reply:
Fixed - let me know if this is not what you were thinking.